wls-basisdaten-service/src/main/java/de/muenchen/oss/wahllokalsystem/basisdatenservice/services/kopfdaten/InitializeKopfdaten.java

```diff
@@ -0,0 +1,62 @@
+package de.muenchen.oss.wahllokalsystem.basisdatenservice.services.kopfdaten;
+
+import de.muenchen.oss.wahllokalsystem.basisdatenservice.exception.ExceptionConstants;
+import de.muenchen.oss.wahllokalsystem.basisdatenservice.services.wahlen.WahlModel;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.util.ExceptionFactory;
+import de.muenchen.oss.wahllokalsystem.wls.common.security.domain.BezirkUndWahlID;
+import lombok.RequiredArgsConstructor;
+import lombok.val;
+
+@RequiredArgsConstructor
+public class InitializeKopfdaten {
+
+    private final ExceptionFactory exceptionFactory;
+
+    protected void initKopfdaten(BasisdatenModel basisdatenModel) {
```

Comment:
Wann/wo wird die Methode für das initialize Objekt aufgerufen? Oder wird diese Methode überhaupt aufgerufen?

Reply:
In der Tat jetzt nicht, aber im nächsten Issue mit "Wahltermindaten".